docs/content/latest/yugabyte-cloud/cloud-examples/_index.md

```diff
@@ -1,27 +1,23 @@
 ---
-title: Develop
-linkTitle: Develop
-description: Get started developing applications for Yugabyte Cloud.
-headcontent: Get started developing applications for Yugabyte Cloud.
+title: Example applications
+linkTitle: Example applications
+description: Example Spring and GraphQL applications for Yugabyte Cloud.
```

Comment:
I would just say "Example applications for Yugabyte Cloud".

You're right that we only have Spring and GraphQL examples as of now but many more are planned to be added in the future.